File: lua/1043.lua
local a = io.read("*n")
local b = io.read("*n")
local c = io.read("*n")

if math.abs(b - c) < a and a < b + c and math.abs(a - c) < b and b < a + c and math.abs(a - b) < c and c < a + b then
    local r = a + b + c
    print(string.format("Perimetro = %.1f", r))
else
    local r = (a + b) * c / 2
    print(string.format("Area = %.1f", r))
end

File: go/1043.go
package main

import (
	"fmt"
	"math"
)

func main() {
	var a, b, c, r float64
	fmt.Scanf("%f", &a)
	fmt.Scanf("%f", &b)
	fmt.Scanf("%f", &c)

	if math.Abs(b-c) < a && a < b+c && math.Abs(a-c) < b && b < a+c && math.Abs(a-b) < c && c < a+b {
		r = a + b + c
		fmt.Printf("Perimetro = %.1f\n", r)
		return
	}

	r = (a + b) * c / 2
	fmt.Printf("Area = %.1f\n", r)
}
